functions/nvm.fish

```diff
@@ -187,6 +197,11 @@ function _nvm_use
         command mv -f $target/$name/$name $nvm_config/$ver.
         command rm -rf $target
         command mv -f $nvm_config/$ver. $target
+
+       if test "$os" = "win"
+           command mkdir $nvm_config/$ver/bin
+           command mv -f $nvm_config/$ver/{node.exe,npm,node_modules} $nvm_config/$ver/bin
+       end
```

Comment:
@Skehmatics Do you mind explaining what's going on here and why do we need it? :)

Reply:
Sure! The Linux and macOS archives put all their binaries in /bin, while the windows archive only contains binaries (although `npm` is a special case, as it is just a wrapper to `npm-cli.js` inside the node modules folder)

Rather than redesign the PATH export to pick and choose what to export, this snippit stuffs those pieces into the /bin folder the script already expects

Reply:
The default code path that runs for other OSes also runs for "win". Is that normal?

Reply:
Yes. This puts the win archive file structure in conformance with that code path

Reply:
New commit changes this from a list to any executable (which will be `node.exe`, `npm`, `npx` (if present), and `node_modules`), but performs towards the same goal.